packages/edit-post/src/editor.native.js

```diff
@@ -88,14 +85,6 @@ class Editor extends Component {
 	componentDidMount() {
 		const { editEntityRecord, postType, postId } = this.props;
 
-		this.subscriptionParentSetFocusOnTitle = subscribeSetFocusOnTitle(
-			() => {
-				if ( this.postTitleRef ) {
```

Comment:
Looks like `postTitleRef` was only being used here, so I'm wondering if we could also remove the references from the following parts:

https://github.com/WordPress/gutenberg/blob/4f3b456bf84bc7e2373898d72028c6877261d003/packages/edit-post/src/editor.native.js#L109-L111

https://github.com/WordPress/gutenberg/blob/4f3b456bf84bc7e2373898d72028c6877261d003/packages/edit-post/src/editor.native.js#L163